source/retryable-writes/tests/supported-single-statement-failure.yml

```diff
@@ -1,59 +1,111 @@
 data:
     - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
 
 minServerVersion: '3.6'
 
 # Each test expects an error due to successive network errors for the first
-# write command. The fail point drops the connection after executing the write
-# command but before its result would be returned, so we still observe changes
-# to the collection. Additionally, our observations assert that the subsequent
-# retry attempt abides exactly-once semantics.
+# write command. Observation of the collection state will assert that the write
+# occurred at-most once, depending on whether the fail point allowed the write
+# to commit before dropping the connection.
 tests:
     -
-        description: "DeleteOne fails after multiple network errors"
+        description: "DeleteOne is committed on first attempt"
```

Comment:
All of the "[Operation] is committed on first attempt" tests in this file do not actually fail. They fail on the first attempt but the retry attempt succeeds.

For others following along, these correspond to single statemate test type 3 in Jeremy's comment here: https://jira.mongodb.org/browse/SPEC-966?focusedCommentId=1696603&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1696603

Reply:
Noted. I believe we can just remove all of these. As-is, it looks like these tests leave the fail point activated after they are done (not really an issue since we instruct drivers to clear fail points after each test).